l1-contracts/bootstrap.sh

```diff
@@ -131,6 +131,84 @@ function gas_report {
   mv gas_report.new.json gas_report.json
 }
 
+function bench {
+  local hash=$(git rev-list -n 1 ${AZTEC_CACHE_COMMIT:-HEAD})
```

Comment:
There's already a content hash defined, we don't need to follow the inferior pattern of git commit